_widget/src/components/app/component.vue

```diff
@@ -139,14 +143,39 @@ export default {
   },
 
   methods: {
-    go (page, params, ignoreHistory) {
-      if (!ignoreHistory)
+    visit (url, event = null) {
```

Comment:
Visit becomes the single path for recently-visited logic, as well as where the article should be displayed.